dockerfiles/drbd-driver-loader/Dockerfile.bionic

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:bionic
 MAINTAINER Roland Kammerer <roland.kammerer@linbit.com>
 
-RUN apt-get update && apt-get upgrade -y && apt-get install -y kmod gnupg wget make gcc patch curl && apt-get clean
+RUN apt-get update && apt-get upgrade -y && apt-get install -y kmod gnupg wget make gcc patch curl dkms && apt-get clean
```

Comment:
I like the clever idea, but I'm not sure if we should pull in all these unnecessary dependencies? hope I got that right, but in a jammy container this is 62 vs. 128 new packages, some of them I'd assume being massive in space

Maybe we can get the gcc versions from something like `apt-cache depends --no-pre-depends --no-recommends --no-suggests --no-conflicts --no-breaks --no-replaces --no-enhances -q=2  dkms`? I hope there is a more pragmatic way actually, because that is horrible to parse, as in:

```
 |Depends: kmod
  Depends: <kldutils>
  Depends: gcc
  Depends: gcc-12
  Depends: dpkg-dev
 |Depends: make
    make-guile
  Depends: build-essential
  Depends: coreutils
  Depends: patch
  Depends: dctrl-tools
```

Reply:
I was looking around for other packages we could use, but apparently, `dkms` is the only one that directly depends on the "right" gcc version:
```
$ apt-cache rdepends gcc-12
  g++-12
  gobjc-12
  gnat-12
  gfortran-12
  gccgo-12
  gcc-12-plugin-dev
  gcc-12-offload-nvptx
  gcc-12-offload-amdgcn
  gcc-12-multilib
  gcc-12-locales
  gcc-12-hppa64-linux-gnu
  g++-12
  dkms
  gobjc-12
  gnat-12
  gfortran-12
  gccgo-12
  gcc-12-plugin-dev
  gcc-12-offload-nvptx
  gcc-12-offload-amdgcn
  gcc-12-multilib
  gcc-12-locales
  gcc-12-hppa64-linux-gnu
```

So I guess we need to get the gcc versions from dkms. Parsing `apt-cache` output looks like fun :sweat:

Reply:
Parsed the output of `apt-get install -s`, as that seems to be manageable.